news/2018-03-18-featured-artist-track-updates.md

```diff
@@ -0,0 +1,244 @@
+</audio>
+
+### Unconquered
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/5/previews/394.mp3" type="audio/mpeg">
+</audio>
+
+### Devourer of All
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/5/previews/391.mp3" type="audio/mpeg">
+</audio>
+
+## dark cat
+
+It turns out that we've had a lingering track sitting nice and snug in the Featured Artist archives since the project's inception, just waiting to be released.
+
+Still just as hot as it was when we first got it, there's now **one** new catstep track for dark cat lovers to feast on.
+
+[Download the new track over at dark cat's Featured Artist listing!](https://osu.ppy.sh/beatmaps/artists/6)
+
+### hot chocolate
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/6/previews/405.mp3" type="audio/mpeg">
+</audio>
+
+## Loki/Thaehan
+
+Back again under the Thaehan alias with his latest EP, *OMGWTF !*, osu!'s favourite EDM master brings **ten** new tracks to the fray, featuring sounds from some much-loved franchises!
+
+[Check out Loki/Thaehan's Featured Artist listing for download links and more information!](https://osu.ppy.sh/beatmaps/artists/7)
+
+### Angry Birds Epic (Remix)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/7/previews/395.mp3" type="audio/mpeg">
+</audio>
+
+### Clown
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/7/previews/396.mp3" type="audio/mpeg">
+</audio>
+
+### Final Boss II
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/7/previews/397.mp3" type="audio/mpeg">
+</audio>
+
+### Hey
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/7/previews/398.mp3" type="audio/mpeg">
+</audio>
+
+### Infinity
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/7/previews/399.mp3" type="audio/mpeg">
+</audio>
+
+### Insert Coin
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/7/previews/400.mp3" type="audio/mpeg">
+</audio>
+
+### Kawaii
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/7/previews/401.mp3" type="audio/mpeg">
+</audio>
+
+### Mega Man 2 Medley
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/7/previews/402.mp3" type="audio/mpeg">
+</audio>
+
+### Overpowered
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/7/previews/403.mp3" type="audio/mpeg">
+</audio>
+
+### Superpower
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/7/previews/404.mp3" type="audio/mpeg">
+</audio>
+
+## Rin/Function Phantom
+
+We're still hard at work trying to acquire the remainder of Rin's absurdly excellent musical lineup, including tracks from his Ocelot alias and the rest of his Function Phantom works.
+
+In addition, there's also over four sets of House mixes we've yet to release, many of which that have been featured on the osulive World Cup streams as backing tracks.
```

Comment:
there are also
osulive World Cup streams -> World Cup livestreams?

Reply:
i dropped this entire paragraph since on review, it doesnt make much sense to announce what we're going to be releasing